python/ray/tune/result.py

```diff
@@ -116,13 +116,22 @@
 STDOUT_FILE = "__stdout_file__"
 STDERR_FILE = "__stderr_file__"
 
+
+def _get_defaults_results_dir() -> str:
```

Comment:
[not related to this PR] in the future, this file and other file like `trial.py` should really move to an execution specific folder rather than under `tune`.